packages/connect/CHANGELOG.md

```diff
@@ -1,9 +1,50 @@
-| Package                          | stable | canary |
-| -------------------------------- | :----: | :----: |
-| npm @trezor/connect              | 9.2.2  | 9.2.3  |
-| npm @trezor/connect-web          | 9.2.2  | 9.2.3  |
-| npm @trezor/connect-webextension | 9.2.2  | 9.2.3  |
-| connect.trezor.io/9              | 9.2.2  |   x    |
+| Package                          | stable |    canary    |
+| -------------------------------- | :----: | :----------: |
+| npm @trezor/connect              | 9.2.2  | 9.2.4-beta.1 |
+| npm @trezor/connect-web          | 9.2.2  | 9.2.4-beta.1 |
+| npm @trezor/connect-webextension | 9.2.2  | 9.2.4-beta.1 |
+| connect.trezor.io/9              | 9.2.2  |      x       |
+
+# 9.2.4-beta.1
+
+-   chore(connect): add new TS3 CA pubkeys and update timestamp (3ae06ac)
+-   fix(connect): BackupDevice now has params (4120912)
+-   test(connect): add changeLanguage test (f5a3207)
+-   chore(connect): remove deprecated code, bump required fw to 1.8.1/2.1.0, fix typo in log, rename firmwareUpdate_v2 to firmwareUpdate (8af325a, 2f14ff6 649a197, b39030c, b67170c)
+-   refactor(connect): popupPromise improvement, pin retries, invalidDeviceState, flattened onCall/inner (320c5a9,6515f13, 7666994, 99ac0e5, 4aa2b46, fa0974d, 9188727, 08a093b, dd6437e, 27d5a44)
+-   fix(connect): intermediary reconnect improved, device authenticity, multi-apps synchronization, reading translations, bin_outputs in txcache (115c718, de1b969, 140ec9a a881142, 88d7608, 0bb13d3)
+-   fix(connect): bin_outputs in txcache (0bb13d3)
+-   feat(connect): update protobuf messages (41bff13)
+
+## connect-popup
+
+-   fix(connect-popup): update text in selectAccount (21f4382)
+-   test(connect-popup): increase general timeout to 60 seconds (f7e3f59)
+-   test(connect-popup): nextra tests for webextension (247d888)
+-   fix(connect-popup): add delay before popup bootstrap to allow contentscript load (00b2056)
+-   fix(connect-popup): webextension example e2e (b9cce02)
+-   test(connect-popup): finetune popup close test (452d3ab)
+-   fix(connect-popup): typo in a comment (8e21eeb)
+-   fix(connect-popup): queue messages sent before init (8850665)
+-   fix(connect-popup): delay popup.js loading to allow content script to init (92d15bc)
+-   fix(connect-popup): wait for POPUP.LOADED in webextension (cb18673)
+-   test(connect-popup): make tests work with nextra explorer (765e5d4)
+
+## connect-explorer
+
+-   use codemirror for json fields, method json params editor, hide setting, sidebar icons (17521ee, 09086d0 6d9845f, d06b4c5, 778b10b)
```

Comment:
maybe something like "complete revamp" could be here :D but I guess we may edit this changelog before stable releases? 

how "well done" should the changelog be written now? are we going to keep it as it is or are we going to edit it before going to stable?